streamx-common/src/main/scala/com/streamxhub/streamx/common/util/FlinkUtils.scala

```diff
@@ -39,4 +42,23 @@ object FlinkUtils {
     }
   }
 
+  def parseDynamicOptions(dynamicParams: String): Array[String] = {
```

Comment:
Maybe it's better?

def parseDynamicOptions(dynamicParams: String): Array[String] = {
    if (StringUtils.isBlank(dynamicParams)) Array.empty[String]; else {
      val matcher: Matcher = ENV_JAVA_OPTS_PATTERN.matcher(dynamicParams)
      val dynamicOptions = ListBuffer[String]()
      val parsedParams = if (!matcher.find) dynamicParams.trim; else {
        dynamicOptions += matcher.group.replace("\"", "")
        dynamicParams.replace(matcher.group, "").trim
      }
      parsedParams.split("\\s+").foreach(x => dynamicOptions += x)
      dynamicOptions.toArray
    }
  }